dataactcore/models/domainModels.py

```diff
@@ -25,4 +25,14 @@ class ObjectClass(Base):
     __tablename__ = "object_class"
     object_class_id = Column(Integer, primary_key=True)
     object_class_code = Column(Text,nullable=False,index=True,unique=True)
-    object_class_name = Column(Text)
\ No newline at end of file
+    object_class_name = Column(Text)
+
+class ProgramActivity(Base):
+    __tablename__ = "program_activity"
+    program_activity_id = Column(Integer, primary_key=True)
+    budget_year = Column(Text,nullable=False)
+    agency_id = Column(Text,nullable=False)
```

Comment:
Or is this agency code being used as part of a TAS-like component in this table? In those cases, it seems like we use the `agency` moniker.